examples/gnrc_minimal/main.c

```diff
@@ -22,16 +22,38 @@
 
 #include "msg.h"
 #include "net/ipv6/addr.h"
+#ifdef MODULE_GNRC_NETIF2
+#include "net/gnrc.h"
+#include "net/gnrc/netif2.h"
+#else
 #include "net/gnrc/netif.h"
 #include "net/gnrc/ipv6/netif.h"
+#endif
 
 int main(void)
 {
-    kernel_pid_t ifs[GNRC_NETIF_NUMOF];
 
     puts("RIOT network stack example application");
 
+#ifdef MODULE_GNRC_NETIF2
+    /* get interface and print their addresses */
+    gnrc_netif2_t *netif = NULL;
+    while ((netif = gnrc_netif2_iter(netif))) {
+        ipv6_addr_t ipv6_addrs[GNRC_NETIF2_IPV6_ADDRS_NUMOF];
+        int res = gnrc_netapi_get(netif->pid, NETOPT_IPV6_ADDR, 0, ipv6_addrs,
+                                  sizeof(ipv6_addrs));
+
+        /* res / 16 (res / sizeof(ipv6_addr_t)) */
+        for (int i = 0; i < (res >> 4); i++) {
```

Comment:
what's  the point in writing `(res >> 4)` when you need to document that?
Just use `(res / sizeof(ipv6_addr_t)`. Every compiler will certainly use a shift for that (: